src/mote/main.py

```diff
@@ -37,7 +37,8 @@
 from mote.modules.late import fetch_recent_meetings
 
 main = Flask(__name__)
-recognition_pattern = "(.*)[\-\.]([0-9]{4}-[0-9]{2}-[0-9]{2})-([0-9]{2}\.[0-9]{2})"
+main.config.from_pyfile("config.py")
+recognition_pattern = main.config["RECOGNIITION_PATTERN"]
```

Comment:
You don't need this line at this point, you can simply use `main.config["RECOGNIITION_PATTERN"]` wherever it's needed directly